test/proto/perf_tests/perf_control.proto

```diff
@@ -138,10 +91,13 @@ message ClientConfig {
   int32 async_client_threads = 7;
   RpcType rpc_type = 8;
   string host = 9;
-  LoadType load_type = 10;
   LoadParams load_params = 11;
 }
 
+message ClientStatus {
+  ClientStats stats = 1;
+}
+
 // Request current stats
 message Mark {
```

Comment:
Does the "Mark" action just get the current stats or it also resets the stats?
Would it make sense to change to add "bool reset" field to indicate if stats should be reset?